### PR TITLE
Use begin end/block instead of query in data commitments

### PR DESCRIPTION
**Pull request by SweeXordious**
Changes the `query` to `begin`/`end` when creating a data root over a set of blocks.

Closes https://github.com/celestiaorg/celestia-core/issues/837 https://github.com/celestiaorg/celestia-app/issues/604

### CODE REVIEW

File: rpc/client/http/http.go [hunk 1]
```diff
@@ -39,24 +39,24 @@ the example for more details.
 
 Example:
 
-		c, err := New("http://192.168.1.10:26657", "/websocket")
```
Comment (SweeXordious) [inline rpc/client/http/http.go:42]: @evan-forbes Can we leave these cosmetics in this PR? or revert them?

File: light/rpc/client.go [hunk 2]
```diff
@@ -27,6 +27,7 @@ var errNegOrZeroHeight = errors.New("negative or zero height")
 type KeyPathFunc func(path string, key []byte) (merkle.KeyPath, error)
 
 // LightClient is an interface that contains functionality needed by Client from the light client.
+//
```
Comment (SweeXordious) [inline light/rpc/client.go:30]:
```suggestion
```

File: rpc/client/http/http.go [hunk 3]
```diff
@@ -39,24 +39,24 @@ the example for more details.
...
-		if err != nil {
-			// handle error
-		}
+	c, err := New("http://192.168.1.10:26657", "/websocket")
+	if err != nil {
+		// handle error
+	}
```
Comment (SweeXordious) [inline rpc/client/http/http.go:45]:
```suggestion
		c, err := New("http://192.168.1.10:26657", "/websocket")
		if err != nil {
			// handle error
		}
```

File: rpc/client/http/http.go [hunk 4]
```diff
@@ -39,24 +39,24 @@ the example for more details.
...
-		if err != nil {
-			// handle error
-		}
-		defer c.Stop()
+	// call Start/Stop if you're subscribing to events
+	err = c.Start()
+	if err != nil {
+		// handle error
+	}
+	defer c.Stop()
```
Comment (SweeXordious) [inline rpc/client/http/http.go:52]:
```suggestion
		// call Start/Stop if you're subscribing to events
		err = c.Start()
		if err != nil {
			// handle error
		}
		defer c.Stop()
```

File: rpc/client/http/http.go [hunk 5]
```diff
@@ -39,24 +39,24 @@ the example for more details.
...
 
-		res, err := c.Status()
-		if err != nil {
-			// handle error
-		}
+	res, err := c.Status()
+	if err != nil {
+		// handle error
+	}
```
Comment (SweeXordious) [inline rpc/client/http/http.go:57]:
```suggestion
		res, err := c.Status()
		if err != nil {
			// handle error
		}
```

File: rpc/client/http/http.go [hunk 6]
```diff
@@ -39,24 +39,24 @@ the example for more details.
...
 
-		// handle result
+	// handle result
```
Comment (SweeXordious) [inline rpc/client/http/http.go:59]:
```suggestion
		// handle result
```

**Review by SweeXordious — COMMENTED** [review]
remove cosmetics changes

File: rpc/client/http/http.go (hunk 1 above)
Comment (SweeXordious) [reply rpc/client/http/http.go]: nvm, removed them. thanks

**Review by evan-forbes — APPROVED** [review]
props for debugging this @SweeXordious !!!

:medal_sports: :100: :fire:

**Pull request merged**